### PR TITLE
Trigger episode record creation

### PR DESCRIPTION
**Pull request by zainm97**
## Description
Front end calls lambda endpoint defined in https://github.com/NHSDigital/Galleri-Invitations/pull/126
Ticket:
https://nhsd-jira.digital.nhs.uk/browse/GAL-695

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: src/app/views/clinic_information/ClinicInformation.js [hunk 1]
```diff
@@ -170,15 +170,16 @@ class ClinicInformation extends Component {
...
         // TODO:Replace api id with latest api id from aws console until we get custom domain name set up
-        "https://6rj4w9o94k.execute-api.eu-west-2.amazonaws.com/dev/calculate-num-to-invite",
+        "https://ptnga5orpe.execute-api.eu-west-2.amazonaws.com/dev/calculate-num-to-invite",
         {
           targetAppsToFill: this.state.appsToFill,
           lsoaCodes: payloadObject
         }
       );
       console.log("logging response = ", response.data)
       this.context.setState({
```
Comment (zainm97) [inline src/app/views/clinic_information/ClinicInformation.js:180]: Only change on this page. Others are API ID updates

File: src/app/views/clinic_summary/ClinicSummary.js
Comment (zainm97) [inline src/app/views/clinic_summary/ClinicSummary.js:1]: API ID updates only here

**Review by Amar-NHS — APPROVED** [review]
Looks good. Approved!

**Review by Abdul-NHS — APPROVED** [review]
lgtm

**Pull request merged**